spec/index.html

```diff
@@ -317,26 +317,29 @@ <h3>Triple Terms and Reification</h3>
     <p>A <a>triple term</a> is an [=RDF triple=] used as an [=RDF term=]
       in the [=object=] position of another triple.</p>
 
-    <p>A <a>triple term</a> is not necessarily asserted, allowing
-      statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
-      This allows statements to be made about relationships that may be contradictory.
-      For a <a>triple term</a> to be asserted,
-      it must also appear in a graph as an <a>asserted triple</a>.</p>
-
-    <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>;
-      such a <a>triple</a> is then a <dfn>reifying triple</dfn>.
-      The <a>subject</a> of that <a>triple</a> is called a <dfn>reifier</dfn>.
-      Assertions on the <a>triple term</a> are made using the <a>reifier</a>.
-      A concrete syntax may provide a special notation for specifying <a>reifying triples</a>.</p>
-
-    <p class="note">Using the <a>reifier</a> as an indirect way of referencing a <a>triple term</a>
-      allows multiple groups of assertions to be separated from each other,
-      which might be necessary if the same <a>triple term</a> was derived from different sources.
-      Assertions are always made using the <a>reifier</a>, which can be the <a>subject</a> or <a>object</a>
-      of different triples.
-      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]],
-      may have shortcuts for capturing a <a>triple term</a> with its <a>reifier</a>.</p>
+    <p>This usage of a <a>triple term</a> only functions as a reference to a
+      <a>proposition</a>, and does not in itself imply its truth. For this
+      triple to also be asserted, it must appear in a graph as an <a>asserted
+      triple</a>. This allows for statements to be made about statements
+      independent of their assertion within an <a>RDF graph</a>.
+    </p>
+
+    <p>
+      When a <a>triple term</a> is used as the object of a <a>triple</a> with
+      the predicate <code>rdf:reifies</code>, such a triple is a
+      <dfn>reifying triple</dfn>.
+      The <a>subject</a> of that triple is called a <dfn>reifier</dfn>.
+      The <a>IRI</a> or <a>blank node</a> denoting a reifier can be the
+      subject or <a>object</a> of other triples.
```

Comment:
```suggestion
    subject or object of other triples.
```
as recently referenced.

Reply:
I think this is better. But I'm unsure about the scope we prefer for when referencing again? It seemed to be almost paragraph-by-paragraph in this section (and `object` is only referenced two paragraphs up). I can do this change and clean up some more recently referenced terms in this section if preferred?

Reply:
Sorry, I missed the previous in-paragraph use! I'll commit this suggestion.

(Maybe some more cleanups are needed, but I'll probably stick to the "same paragraph" pattern unless otherwise advised.)